test/CMakeLists.txt

```diff
@@ -104,7 +104,7 @@ IF(TARGET ROOT::ROOTDataFrame)
 endif()
 
 
-find_package(HepMC3 3.3)
+find_package(HepMC3)
```

Comment:
HepMC3 implies `3.0`, I assume?

Reply:
Almost, there are some alpha and beta versions from 8 years ago but I don't think those are easily installable... so it should be fine
https://gitlab.cern.ch/hepmc/HepMC3/-/tags